rpcserver.go

```diff
@@ -4243,32 +4197,32 @@ func (r *rpcServer) ListPayments(ctx context.Context,
 
 	rpcsLog.Debugf("[ListPayments]")
 
-	payments, err := r.server.chanDB.FetchAllPayments()
-	if err != nil && err != channeldb.ErrNoPaymentsCreated {
+	payments, err := r.server.chanDB.FetchSentPayments()
```

Comment:
Note to self: these are no longer sorted by invoice ID, consider sorting them by creation date.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2761/commits/cf6aa8e2e1cb501358bf06b71b26d299af24ea08